From 4fd06b9825465ae5eb2a9b862cdb89098f655f14 Mon Sep 17 00:00:00 2001
From: Jimi Huotari <chiitoo@gentoo.org>
Date: Sat, 17 Mar 2018 18:16:39 +0200
Subject: [PATCH] frontend-tools: Include 'QAction' to fix build against Qt
 5.11

Some headers are no longer unconditionally included, leading to
build failures.

http://code.qt.io/cgit/qt/qtbase.git/commit/?id=748836dae80a7d11901f3f0630168829076d11a8
http://code.qt.io/cgit/qt/qtbase.git/commit/?id=000c76ada5cc21479fc479be16a7507fed6490f8
http://code.qt.io/cgit/qt/qtbase.git/commit/?id=058474884c2505a8a00d4c59b4922bfcd3597c2f
---
 UI/frontend-plugins/frontend-tools/scripts.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/UI/frontend-plugins/frontend-tools/scripts.cpp b/UI/frontend-plugins/frontend-tools/scripts.cpp
index d923dcf610..364757e48b 100644
--- a/UI/frontend-plugins/frontend-tools/scripts.cpp
+++ b/UI/frontend-plugins/frontend-tools/scripts.cpp
@@ -12,6 +12,7 @@
 #include <QFont>
 #include <QDialogButtonBox>
 #include <QResizeEvent>
+#include <QAction>
 
 #include <obs.hpp>
 #include <obs-module.h>
